dev/tasks/python-wheels/github.osx.yml

```diff
@@ -122,6 +122,11 @@ jobs:
           python-version: 3.12
           update-environment: false
 
+      - name: Install MinIO
+        run: |
+          $(brew --prefix bash)/bin/bash \
+            arrow/ci/scripts/install_minio.sh latest /usr/local
+
```

Comment:
Could you move this to after "Install GCS testbench" or before "Set up Python for GCS testbench"?